airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -155,9 +156,34 @@ def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[
         params.update(**stream_slice)
         return params
 
-    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]):
+    def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iterable[Mapping]:
+        for record in super().parse_response(response, **kwargs):
+            if self.cursor_field in record:
+                record[self.cursor_field] = pendulum.from_timestamp(float(record[self.cursor_field]))
+
+            yield record
+
+    def _cast_cursor_field(self, current_stream_state: MutableMapping[str, Any]):
```

Comment:
why do we need this method? shouldn't we already know & control the type of the cursor field?

Reply:
deleted this function